### PR TITLE
update to 4.0.8

### PR DESCRIPTION
**Pull request by cbouss**
Changes:
- update dependencies
- add test run

https://github.com/jupyterlab/jupyterlab/tree/v4.0.8
https://github.com/jupyterlab/jupyterlab/blob/v4.0.8/pyproject.toml
https://github.com/jupyterlab/jupyterlab/blob/v4.0.8/jupyterlab/staging/package.json#L211

### CODE REVIEW

**Review by M-Waszkiewicz-Anaconda — APPROVED** [review]
LGTM

File: recipe/meta.yaml [hunk 1]
```diff
@@ -28,58 +28,60 @@ app:
 
 requirements:
   host:
-    - jupyter-packaging 0.12.0
-    - packaging
+    - jupyter-packaging
```
Comment (skupr-anaconda) [inline recipe/meta.yaml]: Is jupyter-packaging no more required https://github.com/jupyterlab/jupyterlab/blob/2c9dff7e8343400b5d7d65839cd2f48e8d5ddb69/pyproject.toml#L5

**Review by skupr-anaconda — APPROVED** [review]
LGTM but jupyter-packaging

File: recipe/meta.yaml (hunk 1 above)
Comment (cbouss) [reply recipe/meta.yaml]: I think you are right, I'll test and remove it if not needed. Thanks.

**Review by JeanChristopheMorinPerso — APPROVED** [review]

**Pull request merged**